### DISCUSSION TITLE
REST API docs site returns 503

### DISCUSSION THREAD

**Discussion by BanzaiMan** [community/community#24503] [Apps, API and Webhooks]
<p><aside class="onebox whitelistedgeneric">
  <header class="source">
      <a href="https://developer.github.com/v3/" target="_blank" rel="nofollow noopener">GitHub Developer</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/25666/181085091-54f0e8de-60d3-4b8d-9d08-3ff4bba0350e.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/26361d706a8a255479f0cb328577f99ad8731275_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/2/26361d706a8a255479f0cb328577f99ad8731275_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/2/26361d706a8a255479f0cb328577f99ad8731275_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/26361d706a8a255479f0cb328577f99ad8731275_2_10x10.png">

<h3><a href="https://developer.github.com/v3/" target="_blank" rel="nofollow noopener">GitHub API v3</a></h3>

<p>Get started with one of our guides, or jump straight into the API documentation.</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
<br>
advises us to visit<br>
<a href="https://docs.github.com/v3/" class="onebox" target="_blank" rel="nofollow noopener">https://docs.github.com/v3/</a><br>
but that returns 503:</p>
<pre><code>Error 503 first byte timeout
</code></pre>
<p>From<br>
<a href="https://docs.github.com/en" class="onebox" target="_blank" rel="nofollow noopener">https://docs.github.com/en</a><br>
“Rest API” sends us to<br>
<a href="https://docs.github.com/en/rest" class="onebox" target="_blank" rel="nofollow noopener">https://docs.github.com/en/rest</a></p>
<p>which seems to be the right URL.</p>
<p>(The issue could be transient; while I was composing this message, <a href="https://docs.github.com" rel="nofollow noopener">https://docs.github.com</a> started returning 503 as well.)</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/banzaiman">@banzaiman</a>, thanks for taking the time to report this! We’ve been experiencing some trouble with our documentation site today, but at this point the issue should be resolved. Our apologies for the inconvenience.</p>
